regression/scripts/weights/weights.py

```diff
@@ -19,72 +19,158 @@
 sys.path.append('../Vehicles')
 # the analysis functions
 
-from Boeing_737 import vehicle_setup
-from Cessna_172 import vehicle_setup as vehicle_setup_general_aviation
+from Boeing_737 import vehicle_setup
+from Cessna_172 import vehicle_setup as vehicle_setup_general_aviation
 from BWB import vehicle_setup  as bwb_setup
 from Solar_UAV import vehicle_setup  as hp_setup
 
 
 def main():
-  
-    vehicle = vehicle_setup()    
-    weight = Tube_Wing.empty(vehicle)
-    
-    # regression values    
+
+    vehicle = vehicle_setup()
+    weight = Common.empty_weight(vehicle, method_type = "SUAVE")

```

Comment:
If we're still using the SUAVE method for the below why do the weights here change?

Reply:
Good question. 
The wing and fuselage weight change due to the change of ultimate load factor from 2.5 to the default of 3.75.
The empty weight also changes due to the restructuring of the empty weight where the operational items are only included in the OEW and not in the EW. The systems weight changed because the operational items are not included in the system weight anymore and both tail weight are dependent on the ultimate load factor and thus also change.